### ISSUE TITLE
PipelineFinally bug or loss of function?

### ISSUE THREAD

**Issue by zops** [tektoncd/pipeline#3092]
**PipelineFinally Doc：**
https://tekton.dev/docs/pipelines/pipelines/#specifying-parameters-in-final-tasks

**Issue：**
The support of params is described in the finally document. However, when params is configured in finally, it is found that variable replacement is not supported.
This is not mentioned in the document. I don't know whether variable is not supported or configuration error.

**Example：**
```
spec:
  params:
    - name: a
  tasks:
    - name: tests
      taskRef:
        name: integration-test
      params:
        - name: a
          value: $(params.a)        # valid variable
  finally:
    - name: report-results
      taskRef:
        Name: report-results
      params:
        - name: a
          value: $(params.a)        # invalid variable
```

**Comment by zops** [comment]
Upgrade from v0.14.0 to v0.15.2, the problem is solved

**Comment by bobcatfish** [comment]
Glad you got it worked out @zops, sorry for the confusion! there was a bug with that which was fixed in [0.14.1](https://github.com/tektoncd/pipeline/releases/tag/v0.14.1)
